packages/features/portfolio/src/containers/portfolio-overview/hooks/use-totals-chart.tsx

```diff
@@ -27,115 +27,223 @@ import { FormattedMessage } from 'react-intl';
 export const useTotalsChart = (baseCurrency: FiatKeys) => {
   const windowDimensions = useWindowDimensions();
   const network = useSelectedNetwork();
+  const { isMobileView } = useAppStore();
   const { currentAPY, netWorth, debts, assets } =
     useAccountCurrentFactors(network);
   const [secondsMultiple, setSecondsMultiple] = useState(1.5);
 
-  if (windowDimensions.width <= 1152 && secondsMultiple !== 1.2) {
-    setSecondsMultiple(1.2);
-  }
-  if (windowDimensions.width > 1200 && secondsMultiple !== 1.5) {
-    setSecondsMultiple(1.5);
-  }
+  if (isMobileView) {
```

Comment:
I'm seeing linter errors here because the hooks are being called conditionally, I think we can just make a copy of this file for the mobile portfolio overview?